README.md

```diff
@@ -17,9 +19,15 @@ In order to get this application running on your local development environment,
 1. Fork the repository
 2. Clone your forked repository onto your local machine
 3. inside your CLI (Command Line Interface), move into your working directory
-4. run `npm i` inside the root *and* client folder to install all dependencies needed for this project
+4. run `npm i` inside the root, client *and* server folder to install all dependencies needed for this project
```

Comment:
We should also include the start up and set up scripts that have been merged into dev.  If you're developing on a osx and ubuntu you can run `npm run dev-setup` from the root folder which will install dependencies in root, client, and server.  Afterwards you can run `npm run dev` and it will spin up both the client and server projects for you.